proto/options.proto

```diff
@@ -384,6 +384,11 @@ message Options {
     expansion_type = 6;
   }
 
+  enum ReverseTimeTracking {
+    heuristic = 0;
```

Comment:
maybe prefix the members with `rtt_` or so. a very annoying thing in pbf is that enum members are children of the parent, i.e. this is `Options::disabled` etc, where `disabled` is a bit generic for the whole `Options` object